ms-windows/templates/1.1.1/x64/openssl-light.iss

```diff
@@ -0,0 +1,148 @@
+; Copyright 2016-2019 The OpenSSL Project Authors. All Rights Reserved.
+;
+; Licensed under the OpenSSL license (the "License").  You may not use
+; this file except in compliance with the License.  You can obtain a copy
+; in the file LICENSE in the source distribution or at
+; https://www.openssl.org/source/license.html
+
+[Setup]
+AppName=OpenSSL Light (64-bit)
+AppVersion=[[DOTTED_VERSION]]
+AppVerName=OpenSSL [[DOTTED_VERSION]] Light (64-bit)

```

Comment:
I am not sure `OpenSSL Light` is right name for the installation. Perhaps `OpenSSL Runtime` vs. `OpenSSL Full` would be better?

Reply:
@t8m That one bothers me too.

However, `Runtime` kind of implies that it contains only `.dll` files without the `openssl.exe` binary.

How about `OpenSSL User Setup` (binaries only) and `OpenSSL Developer Setup` (full version with docs/headers/libs)?

That IMO conveys the intent more clearly.